src/Apps/Conversation/Components/DetailsHeader.tsx

```diff
@@ -21,15 +23,23 @@ interface DetailsHeaderProps extends DetailsProps {}
 
 export const DetailsHeader: FC<DetailsHeaderProps> = props => {
   const { showDetails, setShowDetails } = props
+  const isMobile =
+    __internal__useMatchMedia(themeProps.mediaQueries.xs) ||
+    __internal__useMatchMedia(themeProps.mediaQueries.sm)
+
+  if (isMobile) {
+    return null
+  }
+
   return (
     <StackableBorderBox
       p={0}
       flexDirection="column"
       width={showDetails ? "376px" : "0"}
       maxWidth={showDetails ? "auto" : "0"}
-      display={["none", "none", "flex", "flex", "flex"]}
       {...props}
       borderTop="none !important"
+      borderBottom="none !important"
```

Comment:
I wonder if we have to do this if the `StackableBorderBox` props are spread/used correctly on Palette

Reply:
Have you tried `borderBottom: "0px"` ?